conda-store-server/conda_store_server/api.py

```diff
@@ -142,7 +146,7 @@ def list_conda_packages(db, search: str = None):
     filters = []
     if search:
         filters.append(orm.CondaPackage.name.contains(search, autoescape=True))
-    return db.query(orm.CondaPackage).filter(*filters)
+    return db.query(orm.CondaPackage).join(orm.CondaChannel).filter(*filters)
```

Comment:
Perfect. I believe (could be wrong) that sqlalchemy will optimize out the `join` if `CondaChannel` is not used. But either way I think this a great to add.

Reply:
done.